src/stormpath.config.js

```diff
@@ -169,6 +186,24 @@ angular.module('stormpath.CONFIG',[])
     /**
     * @ngdoc property
     *
+    * @name OAUTH_PROVIDERS_ENDPOINT
+    *
+    * @propertyOf stormpath.STORMPATH_CONFIG:STORMPATH_CONFIG
+    *
+    * @description
+    *
+    * Default: `/oauth/provider`
+    *
+    * The endpoint that is used for fetching a list of available OAuth providers.
+    * Used by {@link stormpath.socialLoginService.$socialLogin#getProviders $socialLogin.getProviders()}.
+    *
+    */
+    OAUTH_PROVIDERS_ENDPOINT: '/oauth/providers',
```

Comment:
How about `/spa-config/social-providers` so that we can keep the code as it is without generalising too much?